packages/platform-sdk-support/src/bignumber.ts

```diff
@@ -133,6 +133,17 @@ export class BigNumber {
 		return BigNumber.make(this.#value.multipliedBy(this.toBigNumber(value)));
 	}
 
+	/**
+	 * Returns the sum of the different values.
+	 *
+	 * @param {NumberLike[]} values
+	 * @returns {string}
```

Comment:
This method returns BigNumber.

Reply:
Fixed.. Having to write types *twice* is probably very error prone